addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -47,7 +47,7 @@ function animateClone($cart, $elem, offsetTop, offsetLeft) {
     });
     return new Promise(function (resolve, reject) {
         var $imgtodrag = $elem.find('img').eq(0);
-        if ($imgtodrag.length) {
+        if ($imgtodrag.length && $cart.length) {
```

Comment:
This check should be the first line of the method, the whole method makes no sense if there is no $cart
Something like
```js
if (!$cart.length) {
    return Promise.resolve();
}
```